### PR TITLE
chore: add se-ui

### PR DESCRIPTION
**Pull request by kelvah**
Opening this PR for the Smart Events application. We need the creation of a corresponding `se-ui-build` repo.
The UI will be a federated module loaded inside Application Services (like cos, kas and others).

Could you please add as admin also @manstis and @vpellegrino?

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -756,6 +756,9 @@ application-services:
       - id: service-registry
         title: "Service Registry"
         group: overview
+      - id: smart-events
```
Comment (Hyperkid123) [inline main.yml:786]: main.yml is only used to upload assets to Akamai. It no longer handles app location or navigation. You have to register a new FE module. Please follow these steps: https://github.com/RedHatInsights/cloud-services-config#chromefed-modulesjson

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

**Comment by kelvah** [comment]
I'm closing this PR for now. We'll resume this task later on. Thank you.

File: main.yml (hunk 1 above)
Comment (kelvah) [reply main.yml:786]: @Hyperkid123 I've added the navigation link to `application-services-navigation.json`

**Review by Hyperkid123 — APPROVED** [review]

**Comment by kelvah** [comment]
Hi @Hyperkid123, just to be sure, is there anything else I should do to request the setup of the `se-ui-build` repo? Or should I just wait?
Thank you

**Comment by Hyperkid123** [comment]
> Hi @Hyperkid123, just to be sure, is there anything else I should do to request the setup of the `se-ui-build` repo? Or should I just wait? Thank you

Nope, I think you are good to go.

**Pull request merged**